aten/src/ATen/native/BinaryOps.cpp

```diff
@@ -622,5 +648,9 @@ Tensor& fmod_(Tensor& self, Scalar other) {
   return at::fmod_out(self, self, other);
 }
 
+Tensor true_divide(const Tensor& self, Scalar divisor) {
```

Comment:
any reason not to put this with the rest of the true_divide family?

Reply:
It's just the file's current convention/structure to put the variants that accept scalars after those that accept tensors. The same split is evident for the div implementation, for example. We don't have to follow it.

Reply:
nah, no need to change.